Amplify/DevMenu/Data/LogEntryHelper.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Helper class to fetch log entry related information
+struct LogEntryHelper {
+
+    /// Helper function to get current time in a specified format
+    static func getDateFormatter() -> DateFormatter {
```

Comment:
Better move the dateFormatter into a static variable, otherwise this object will be created as many times as there are rows in the table. [ Apple doc: Cache formatters for efficiency](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/DataFormatting/Articles/dfDateFormatting10_4.html#//apple_ref/doc/uid/TP40002369-SW10). Something like this:

```swift
private static let dateFormatter: DateFormatter = {
        let f = DateFormatter()
        f.dateFormat = "yyyy-MM-dd HH:mm:ss.SSSS"
        return f
 }()

static func dateString(from date: Date) -> String {
        return dateFormatter.string(from: date)
}
```